.github/workflows/lint.yml

```
@@ -0,0 +1,24 @@
+    defaults:
+      run:
+        working-directory: backend
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Run linter
+        run: npm run lint
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)